spec/select_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'select.rb'
+
+describe Select do
+		let(:burger){double (:burger)}
+		it 'shows the menu' do 
+			men = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22,'Chips'=> 3.25}
+			expect(subject.menu).to eq men
```

Comment:
For anywhere you have subject in your testing, you can set up described_class -- 
https://github.com/makersacademy/airport_challenge/blob/master/docs/review.md#use-named-subject-with-described_class